mesonbuild/interpreter/type_checking.py

```diff
@@ -56,28 +55,28 @@ def _install_mode_validator(mode: T.List[T.Union[str, bool, int]]) -> T.Optional
 
     perms = mode[0]
     if not isinstance(perms, (str, bool)):
-        return 'permissions part must be a string or false'
+        return 'permissions string must be a string or false'
```

Comment:
Permissions ***string*** must be either the aforementioned string, or else violate that rule and be something which is not a string?

Reply:
Argh, I'll reword.